search-commons/src/test/java/no/unit/nva/search/SearchClientTest.java

```diff
@@ -102,27 +94,34 @@ public SearchResponse search(SearchRequest searchRequest, RequestOptions request
         SearchClient searchClient = new SearchClient(restClientWrapper);
         int resultSize = 1 + randomInteger(1000);
         searchClient.findResourcesForOrganizationIds(getSampleViewingScope(),
-                                                     resultSize,
+                                                     resultSize, 0,
                                                      ELASTICSEARCH_ENDPOINT_INDEX);
         var sentRequest = sentRequestBuffer.get();
         var actualRequestedSize = sentRequest.source().size();
         assertThat(actualRequestedSize, is(equalTo(resultSize)));
     }
 
-    private ViewingScope getSampleViewingScope() {
-        ViewingScope viewingScope = new ViewingScope();
-        viewingScope.setIncludedUnits(Set.of(randomUri(), randomUri()));
-        viewingScope.setExcludedUnits(Set.of(randomUri()));
-        return viewingScope;
-    }
+    @Test
+    void shouldReturnResponseWithResultsFromDefinedFromWhenSearchingForResources() throws ApiGatewayException {
```

Comment:
This test name and the the test name of the test before is misleading. We test only what is sent to ES, not what ES is replying to us.